docs-ref-services/key-vault.md

```diff
@@ -1,138 +1,107 @@
 ---
 title: Azure Key Vault libraries for Python
 description: Reference documentation for the Python client libraries for Azure Key Vault
-author: sptramer
-manager: carmonm
-ms.author: sttramer
-ms.date: 06/10/2019
+author: kraigb
+manager: barbkess
+ms.author: kraigb
+ms.date: 03/10/2020
 ms.topic: conceptual
 ms.devlang: python
 ms.service: key-vault
 ---
 
 # Azure Key Vault libraries for Python
 
-[Azure Key Vault](/azure/key-vault/) is Azure's storage and management system for cryptographic keys, secrets, and certificate
-management. The Python SDK API for Key Vault is split between client libraries and management libraries.
-
-Use the client library to:
-- Access, update, or delete items stored in an Azure Key Vault
-- Get metadata for stored certificates
-- Verify signatures against symmetric keys in Key Vault
+[Azure Key Vault](/azure/key-vault/) provides secure cloud storage on Azure for cryptographic keys, secrets, and certificates. The Python SDK API for Key Vault provides both management and client libraries.
 
 Use the management library to:
+
 - Create, update, or delete new Key Vault stores
 - Control vault access policies
 - List vaults by subscription or resource group
 - Check for vault name availability
 
-## Install the libraries
-
-### Client library
+Use the client library to:
 
-```bash
-pip install azure-keyvault
-```
+- Access, update, or delete items stored in an Azure Key Vault
+- Get metadata for stored certificates
+- Verify signatures against symmetric keys in Key Vault
```

Comment:
Do you mean *asymmetric* keys here? Key Vault's cryptographic key API doesn't presently support symmetric keys.

Reply:
Probably--this is old content that I didn't originally author, so it's probably an old typo.